rpc/webhooks.go

```diff
@@ -29,3 +36,14 @@ func (r *rpc) Webhooks(ctx *gin.Context) {
 
 	ctx.Status(http.StatusOK)
 }
+
+// Function that iterates over array of repo subString urls
+func excludeRepo(excludeRepo []string, repoUrl string) bool {
+	for _, repoUrlSubstr := range excludeRepo {
+
+		return strings.Contains(repoUrl, repoUrlSubstr)
+
+	}
+
+	return false
+}
```

Comment:
```suggestion
func arrayContains(arr []string, str string) bool {
	for _, a := range are {
	       if a == str {
	              return true
	       }
	}

	return false
}
```

Reply:
If you want to be extra safe with this one it may be worth `strings.ToLower`(ing) them both